triage/summarize/cluster.go

```diff
@@ -68,7 +70,7 @@ func clusterLocal(failuresByTest failuresGroup, numWorkers int) nestedFailuresGr
 
 	numFailures := 0 // The number of failures processed so far
 	start := time.Now()
-	logInfo("Clustering failures for %d unique tests...", len(failuresByTest))
+	klog.V(2).Infof("Clustering failures for %d unique tests...", len(failuresByTest))
```

Comment:
What is enabling the V-level logging? Will it log nothing if not enabled? https://godoc.org/github.com/kubernetes/klog#Verbose.Enabled

Reply:
I think by default it logs at V2, but regardless it's being set here:  
https://github.com/kubernetes/test-infra/pull/18858/files#diff-e209704c6b63bec066319227a3d98099R94